tools/scenario-player/scenario_player/main.py

```diff
@@ -0,0 +1,163 @@
+import json
+import logging
+import traceback
+from datetime import datetime
+from os.path import basename
+
+import click
+import gevent
+import structlog
+from blessings import Terminal
+from eth_utils import to_checksum_address
+from web3.utils.transactions import TRANSACTION_DEFAULTS
+
+from raiden.accounts import Account
+from raiden.log_config import configure_logging
+from scenario_player.runner import ScenarioRunner
+from scenario_player.tasks import TaskState
+from scenario_player.utils import DummyStream, LogBuffer, send_notification_mail
+
+log = structlog.get_logger(__name__)
+
+TRANSACTION_DEFAULTS['gas'] = lambda web3, tx: web3.eth.estimateGas(tx) * 2
```

Comment:
Why do we need this? Are the estimates to low?

Reply:
Ah no, that's an old leftover. Good catch. I'll remove it.